lib/active_triples/application_strategy.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveTriples
+  ##
+  # Default application strategy which just copies all configured properties
+  # from a data property to a new resource, assuming it supports the #property
+  # interface.
+  class ApplicationStrategy
```

Comment:
Can we think of a better name for this? I think you are saying this a strategy for "applying" schemas to resources, but "Application" is so overloaded in our field. How about EmploymentStragegy? AccessionStrategy? ExtensionStrategy?